parsing/printast.ml

```diff
@@ -887,7 +886,8 @@ and directive_argument i ppf x =
   match x with
   | Pdir_none -> line i ppf "Pdir_none\n"
   | Pdir_string (s) -> line i ppf "Pdir_string \"%s\"\n" s;
-  | Pdir_int (i) -> line i ppf "Pdir_int %d\n" i;
```

Comment:
There was an error here because the argument `i` of the function was shadowed by the pattern matching,